warehouse/query-core/src/main/java/datawave/query/iterator/ivarator/IvaratorCacheDirConfig.java

```diff
@@ -34,6 +34,9 @@ public class IvaratorCacheDirConfig {
     // the minimum percent of available storage required to use this filesystem
     final protected double minAvailableStoragePercent;
     
+    // Do we require the ivarators to cleanup after completion (true by default)
+    final protected boolean requiresCleanup;
```

Comment:
Are we ever realistically going to set this to false?

Reply:
I wanted to leave the possibility of turning if off in case we could the code was deleting stuff if should not be deleting.  This might allow us to keep the system running until the bug is uncovered.  Just trying to be safe.